{{cookiecutter.project_shortname}}/invenio.cfg

```diff
@@ -0,0 +1,130 @@
+#: Librarians email for internal system notifications.
+ILS_MAIL_NOTIFY_MANAGEMENT_RECIPIENTS = ["librarians@{{ cookiecutter.project_site }}"]
+#: Disable email sending by default.
+MAIL_SUPPRESS_SEND = True
+#: Email address for email notification sender.
+MAIL_NOTIFY_SENDER = "librarians@{{ cookiecutter.project_site }}"
+#: Email address used as sender of account registration emails.
+SECURITY_EMAIL_SENDER = SUPPORT_EMAIL
+#: Email subject for account registration emails.
+SECURITY_EMAIL_SUBJECT_REGISTER = _("Welcome to the {{ cookiecutter.project_name }}")
+
+###############################################################################
+# MAILS templates
+###############################################################################
+ILS_NOTIFICATIONS_TEMPLATES_CIRCULATION = {
+    "bulk_extend": "{{ cookiecutter.project_shortname }}_bulk_extend.html",
+    "cancel": "{{ cookiecutter.project_shortname }}_cancel.html",
+    "request": "{{ cookiecutter.project_shortname }}_request.html",
+    "request_no_items": "{{ cookiecutter.project_shortname }}_request_no_items.html",
+    "checkin": "{{ cookiecutter.project_shortname }}_checkin.html",
+    "checkout": "{{ cookiecutter.project_shortname }}_checkout.html",
+    "extend": "{{ cookiecutter.project_shortname }}_extend.html",
+    "overdue_reminder": "{{ cookiecutter.project_shortname }}_overdue_reminder.html",
+    "expiring_reminder": "{{ cookiecutter.project_shortname }}_will_expire_in_reminder.html",
+}
+
+ILS_NOTIFICATIONS_TEMPLATES_DOCUMENT_REQUEST = {
+    "request_accepted": "{{ cookiecutter.project_shortname }}_document_request_accept.html",
+    "request_declined_user_cancel":
+        "{{ cookiecutter.project_shortname }}_document_request_decline_user_cancel.html",
+    "request_declined_in_catalog":
+        "{{ cookiecutter.project_shortname }}_document_request_decline_in_catalog.html",
+    "request_declined_not_found": "{{ cookiecutter.project_shortname }}_document_request_decline_not_found.html",
+    "request_declined_other": "{{ cookiecutter.project_shortname }}_document_request_decline_other.html",
+}
+
+ILS_ILL_NOTIFICATIONS_TEMPLATES = {
+    "extension_accepted": "{{ cookiecutter.project_shortname }}_patron_loan_extension_accept.html",
+    "extension_declined": "{{ cookiecutter.project_shortname }}_patron_loan_extension_decline.html",
+    "extension_requested": "{{ cookiecutter.project_shortname }}_patron_loan_extension_request.html",
+}
+
+ILS_NOTIFICATIONS_TEMPLATES = {"footer": "{{ cookiecutter.project_shortname }}_footer.html"}
+
+###############################################################################
+# Data model extensions
+###############################################################################
+ILS_RECORDS_METADATA_EXTENSIONS = {}
+
+###############################################################################
+# Vocabularies
+###############################################################################
+#ILS_VOCABULARIES = []
```

Comment:
question: is this correct?

Reply:
Yes, it's the name of the variable where user will put his list of available vocabularies